src/components/Class/redux/reducer.js

```diff
@@ -56,6 +56,15 @@ export default (state = initialState, action) => {
           data: action.data,
         },
       };
+    case types.GET_UPDATED_DROP_OUT_CLASSES:
+      return {
```

Comment:
do the data modification here by accessing the data from state

Reply:
https://github.com/navgurukul/bhanwari-devi/pull/85/commits/9feab343ff61492a0305dafdf53c22262ccd9e11